common/src/main/java/whocraft/tardis_refined/client/TardisClientData.java

```diff
@@ -118,6 +141,18 @@ public void tickClientside() {
             landingTime++;
             takeOffTime = 0;
         }
+
+
+        // Responsible for screen-shake. Not sure of a better solution at this point in time.
+        if (isInDangerZone || isCrashing) {
+            if (Minecraft.getInstance().player.level.dimension() == levelKey) {
+                var player = Minecraft.getInstance().player;
+                player.setXRot(player.getXRot() + (player.getRandom().nextFloat() - 0.5f) * flightShakeScale);
+                player.setYHeadRot(player.getYHeadRot() + (player.getRandom().nextFloat() - 0.5f) * flightShakeScale);
+            }
+        }
```

Comment:
Could add a client config option to toggle this. 

Also, perhaps we can consider adding a field that defines how many ticks we should shake the screen, and use that to control when we should shake the screen. This way it'll be more flexible for future usecases where we also want to shake the screen.

Reply:
This could be good for the future, but it might be a bit of additional complexity that this task might not need. If you're happy we could move this to another task.